### PR TITLE
adding files for building conda package

### PR DESCRIPTION
**Pull request by dgergel**
The .yaml file still needs a bit more editing, but after that it should be ready for building a conda package.

### CODE REVIEW

File: tonic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package:
+  name: tonic
+  version: "0.13.1"
```
Comment (jhamman) [inline tonic/meta.yaml]: tonic is still at version 0.0.0

File: tonic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+source:
+  fn: tonic-0.13.1.tar.gz
+  url: https://pypi.python.org/packages/source/p/tonic/tonic-0.13.1.tar.gz
```
Comment (jhamman) [inline tonic/meta.yaml]: I'm fairly certain this isn't the right url.  I don't have any memory of adding `tonic` to pypi. Can we checkout from github?

File: tonic/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: https://github.com/joerick/pyinstrument
```
Comment (jhamman) [inline tonic/meta.yaml]: huh?  probably just point to https://github.com/UW-Hydro/tonic

**Comment by jhamman** [comment]
A few more changes are needed.  Can we put the three added files in a `conda-recipe` directory? Also, do you know how to trigger the binstar build?

File: tonic/meta.yaml (hunk 2 above)
Comment (dgergel) [inline tonic/meta.yaml]: @jhamman we can do that, i saw that as an option.

File: tonic/meta.yaml (hunk 3 above)
Comment (dgergel) [inline tonic/meta.yaml]: yup that was a mistake

File: tonic/meta.yaml (hunk 2 above)
Comment (jhamman) [inline tonic/meta.yaml]: okay, for now - let's pull from github.

**Comment by dgergel** [comment]
Yup! Just did that. To trigger the binstar build: 
1) conda build tonic 
2) conda install --use-local tonic 
3)conda convert --platform all path_that_tonic_was_installed_to
4) binstar login (must already have anaconda.org account) 
5) binstar upload ~/miniconda/path_from_step_3_above

I *think* that to trigger the build, the three conda recipe files will need to be out of the conda-recipe directory and in the tonic/tonic directory, but I could be wrong.

**Comment by jhamman** [comment]
Thanks, I'll give it a go here shortly.

**Comment by jhamman** [comment]
Thanks: closing in favor of https://github.com/UW-Hydro/conda-recipes-uwhydro/pull/2

**Pull request closed without merging**